### PR TITLE
Implement unit conversion

### PR DESCRIPTION
**Pull request by RoaldL**
Resolves #144017047

### CODE REVIEW

File: src/smif/convert/__init__.py [hunk 1]
```diff
@@ -112,3 +114,52 @@ def _convert_intervals(self, data, from_temporal, to_temporal):
...
                                         from_temporal, to_temporal)
         return converted
+
+
+class UnitConvertor(object):
+    """Handles the conversion of units for a numpy array of values
+
+    Arguments
+    ---------
+    unit_register: :class:`smif.convert.unit.UnitRegister`
```
Comment (willu47) [inline src/smif/convert/__init__.py]: There are no arguments here.  Can safely remove this.

File: src/smif/convert/__init__.py [hunk 2]
```diff
@@ -112,3 +114,52 @@ def _convert_intervals(self, data, from_temporal, to_temporal):
...
+        A fully populated register of the units that exist in the model
+    """
+    def __init__(self):
+        self.logger = logging.getLogger(__name__)
+        self.units = get_unit_register()
+
+    def convert(self, data, from_unit, to_unit):
+        """Convert the data from set of regions and intervals to another unit
```
Comment (willu47) [inline src/smif/convert/__init__.py]: Need to update this docstring to units

File: src/smif/convert/__init__.py (hunk 1 above)
Comment (RoaldL) [reply src/smif/convert/__init__.py]: Done

File: src/smif/convert/__init__.py (hunk 2 above)
Comment (RoaldL) [reply src/smif/convert/__init__.py]: This is up to date

**Pull request merged**